tensornetwork/backends/tensorflow/tensorflow_backend_test.py

```diff
@@ -257,3 +257,33 @@ def test_multiply(a, b, expected):
   tensor2 = backend.convert_to_tensor(b)
 
   np.testing.assert_allclose(backend.multiply(tensor1, tensor2), expected)
+
+
+@pytest.mark.parametrize("a, b, expected",
```

Comment:
please test this for all supported dtypes

Reply:
Add test cases. Please take another look.